src/ui/properties/ParticleNodeEditor.js

```diff
@@ -0,0 +1,333 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import NodeEditor from "./NodeEditor";
+import ColorInput from "../inputs/ColorInput";
+import InputGroup from "../inputs/InputGroup";
+import ImageInput from "../inputs/ImageInput";
+import CompoundNumericInput from "../inputs/CompoundNumericInput";
+import NumericInputGroup from "../inputs/NumericInputGroup";
+import Vector3Input from "../inputs/Vector3Input";
+import SelectInput from "../inputs/SelectInput";
+
+const ColorCurveOptions = [
+  {
+    label: "Even",
+    value: "Even"
+  },
+  {
+    label: "Ease-in",
+    value: "Ease-in"
+  },
+  {
+    label: "Ease-out",
+    value: "Ease-out"
+  },
+  {
+    label: "Ease-in,out",
+    value: "Ease-in,out"
```

Comment:
I'd change the values of the color curve to use [pascal case](https://stackoverflow.com/a/41769355) and match the function names in ParticleNode.

Ex:
```
value: "EaseInOut"
```